components/global/PrimaryHeader/index.jsx

```diff
@@ -16,7 +16,7 @@ export default function Header({ closeUrl = "/", backgroundColor, route }) {
     onResize: ({ height }) => {
       document.documentElement.style.setProperty(
         "--main-header-height",
-        `${ref.current.offsetHeight}px`
+        `${height}px`
```

Comment:
changed each instance of the `useResizeObserver` to use the `height` value provided by the callback method instead of the `ref` object which would sometimes be null.